velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -846,8 +875,9 @@ JoinFuzzer::PlanWithSplits JoinFuzzer::makeNestedLoopJoinPlan(
     const std::vector<std::string>& outputColumns,
     bool withFilter) {
   auto planNodeIdGenerator = std::make_shared<core::PlanNodeIdGenerator>();
-  const std::string filter =
-      withFilter ? makeJoinFilter(probeKeys, buildKeys) : "";
+  const std::string filter = withFilter
+      ? makeJoinFilter(probeKeys, buildKeys, probeInput, buildInput)
```

Comment:
Let's check out all the methods in this file that create join query plans. We need to test joins with filters not only for nested loop joins.

Reply:
Got it.